release notes/v0.43.0.md

```diff
@@ -0,0 +1,124 @@
+k6 v0.43.0 is here! :tada:
+
+Notable changes in this release include:
+- [xk6-browser](https://github.com/grafana/xk6-browser) is now bundled in k6 as an experimental module, and usable without a separate binary or compilation step!
+- Native support for JavaScript's `async`/`await`.
+- A new experimental module for distributed tracing.
+- Large refactoring of core components to simplify the code base, improve maintainability, and increase test coverage.
+- Bug fixes, UX improvements, and maintenance.
+
+Keep reading for the details.
+
+## Breaking changes
+
+- [#2807](https://github.com/grafana/k6/pull/2807) Use non-zero exit codes for tests aborted by Ctrl+C or the REST API.
+
+  Aborting a test run with Ctrl+C will now exit with code `105`, and stopping via the REST API will exit with code `103`.
+
+
+## New Features
+
+### xk6-browser is now a built-in module [#2884](https://github.com/grafana/k6/pull/2884)
```

Comment:
@andrewslotin @inancgumus @ka3de @ankur22 Does it also make sense to say here that we will now refer to xk6-browser as k6 browser module or simply k6 browser? 🤔

Reply:
@mdcruz We don't need to adjust this heading since it already means that the browser was an extension, and now it's a module. We can continue to refer to it as the k6 browser internally. Still, when it comes to external communications and product positioning, it should be referred to as the browser module, as we discussed. Thanks for pointing this out :)

Reply:
@markjmeier @dgzlopes @sniku, WDYT?